src/transformers/models/llama/modeling_llama.py

```diff
@@ -79,9 +84,12 @@ def __init__(self, config: LlamaConfig, device=None):
         self.original_max_seq_len = config.max_position_embeddings
 
         self.config = config
-        self.rope_init_fn = ROPE_INIT_FUNCTIONS[self.rope_type]
+        if self.rope_type == "default":
+            self.rope_init_fn = _compute_default_rope_parameters
+        else:
+            self.rope_init_fn = ROPE_INIT_FUNCTIONS[self.rope_type]
```

Comment:
hey! the idea is not to impor new fucntion, but to explicitly write what the default is! SO `torch.arange(......)` 😉